params/config.go

```diff
@@ -197,6 +202,9 @@ type ChainConfig struct {
 	// QIP714Block implements the permissions related changes
 	QIP714Block *big.Int `json:"qip714Block,omitempty"`
 	MaxCodeSizeChangeBlock *big.Int `json:"maxCodeSizeChangeBlock,omitempty"`
```

Comment:
looks like it'll be rejected by `CheckMaxCodeConfigData()` so that'll force us to do the migration if a chain init is needed on an old chain with the deprecated configs.

Reply:
appropriate migration has been added for chains during upgrade